pandas/tests/io/excel/test_readers.py

```diff
@@ -67,6 +60,24 @@ def engine(request):
     return request.param
 
 
+@pytest.mark.parametrize("kwargs", [dict(), dict(engine=None)])
+def test_default_engine(datapath, monkeypatch, kwargs):
+    """
+    A test for default ExcelFile engine value (which is None)
+    """
+    monkeypatch.chdir(datapath("io", "data"))
+    expected = "xlrd"
+
+    result = pd.ExcelFile("blank.xls", **kwargs)
+    assert result.engine == expected
+
+
+def test_bad_engine_raises():
```

Comment:
Was this moved for a particular reason? If not typically better to leave things where they are as it makes the diff smaller and review easier

Reply:
the reason for this is https://github.com/pandas-dev/pandas/pull/26543#issuecomment-497126487. I can revert this change in this PR and create a separate PR just with this change.

Reply:
Ah OK makes sense. Fine to leave here - thanks for clarifying